src/NodeApi/Native/JSNativeApi.cs

```diff
@@ -832,19 +832,39 @@ public static void SetObjectTypeTag(this JSValue thisValue, in napi_type_tag typ
 
     public static unsafe void SetObjectTypeTag(this JSValue thisValue, ref Guid typeGuid)
     {
+        #if NETFRAMEWORK
+        byte[] guidBytes = typeGuid.ToByteArray();
```

Comment:
>ToByteArray [](http://example.com/codeflow?start=36&length=11)

It would be better to avoid memory allocation here.
At some point I was creating struct as a local variable and was copying Guid bytes there. While that code cannot be used here "as-is", but just in case how it looked like: [https://github.com/vmoroz/napi-cs/blob/9c216ef0626b07fbfcd4433ff8d527a3ac830504/NodeApi/JSNative.cs#L844](https://github.com/vmoroz/napi-cs/blob/9c216ef0626b07fbfcd4433ff8d527a3ac830504/NodeApi/JSNative.cs#L844)

Reply:
`MemoryMarshal.CreateReadOnlySpan()` is not available in .NET Framework.

But this works:
```C#
        Guid guid = typeGuid;
        napi_type_tag typeTag;
        long* pGuid = (long*)&guid;
        long* pTag = (long*)&typeTag;
        pTag[0] = pGuid[0];
        pTag[1] = pGuid[1];
```